src/app/components/wrapper-components/local-game-wrapper/local-game-wrapper.component.spec.ts

```diff
@@ -227,41 +233,45 @@ describe('LocalGameWrapperComponent (game phase)', () => {
             expect(testUtils.getGameComponent().hideLastMove).toHaveBeenCalledOnceWith();
         }));
     });
+
     describe('Using AI', () => {
-        it('should disable interactivity when AI is selected without level', async() => {
+
+        async function selectFirstElementOfDropDown(dropDownName: string): Promise<void> {
```

Comment:
hmm si on change l'ordre des éléments ça casse le tout, on pourrait pas plutôt ici dire « sélectionne l'élément avec ce nom » ? (ça itérerait sur les options pour trouver celui dont la value correspond pour s'assurer qu'il existe)